docs/src/SelectList.doc.js

```diff
@@ -20,185 +41,361 @@ card(
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Helper text"
+      description="Helper text should be used when additional description may be required to understand the SelectList. Most commonly, this could be text that is legally required to be displayed, or instructions to fill out a form (e.g. proper formatting). If the text is optional, Tooltip could be used instead."
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`<SelectList
+        id="selectlistexample"
+        onChange={() => {}}
+        options={[
+          {label:'Belgium', value: 'belgium'},
+          {label:'Brazil', value: 'brazil'},
+          {label:'Japan', value: 'japan'},
+          {label:'New Zealand', value: 'newzealand'},
+          {label:'Tanzania', value: 'tanzania'},
+          {label:'United States', value: 'usa'},
+        ]}
+        label='Country'
+        helperText='Pick your favorite country (the one with the best fries)'
+      />`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
 );
 
 card(
-  <Example
-    id="exampleWithError"
-    name="Example: With Error Message"
-    description={`SelectList can display error messages if you'd like.
-    To use our errors, simply pass in an \`errorMessage\` when there is an error present.`}
-    defaultCode={`
-function Example(props) {
-  const [hobby, setHobby] = React.useState('sailing')
-  const hobbies = [
-    {
-      value: "sailing",
-      label: "Sailing"
-    },
-    {
-      value: "skiing",
-      label: "Skiing"
-    },
-    {
-      value: "cycling",
-      label: "Cycling"
-    }
-  ];
-  return (
-    <SelectList
-      id="hobby"
-      name="hobby"
-      errorMessage={ hobby === 'sailing' ? 'Sailing is not a hobby, it is a passion' : null}
-      onChange={({ value }) => setHobby(value)}
-      options={hobbies}
-      placeholder="Select hobby"
-      label="Hobby"
-      value={hobby}
-    />
-  );
-}
-    `}
-  />,
+  <MainSection name="States">
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="lg"
+        title="Default"
+        defaultCode={`<SelectList
+        id="selectlistexample"
+        onChange={() => {}}
+        options={[
+          {label:'Austin', value: 'austin'},
+          {label:'Los Angeles', value: 'la'},
+          {label:'New York', value: 'nyc'},
+          {label:'San Francisco', value: 'sf'},
+          {label:'Seattle', value: 'seattle'},
+        ]}
+        label='City'
+      />`}
+      />
+      <MainSection.Card
+        cardSize="lg"
+        title="Focused"
+        defaultCode={`<SelectList
+        id="selectlistexample"
+        onChange={() => {}}
+        options={[
+          {label:'Austin', value: 'austin'},
+          {label:'Los Angeles', value: 'la'},
+          {label:'New York', value: 'nyc'},
+          {label:'San Francisco', value: 'sf'},
+          {label:'Seattle', value: 'seattle'},
+        ]}
+        label='City'
+        focus
```

Comment:
`focus` isn't a prop on SelectList. I'm not quite sure how to trigger this state without messing with a11y for the user…I'll bring this up in the channel

Reply:
As per our conversation today, I removed the `disabled` and `focus` states and moved `errorMessage` to the variants seciton.